oggm/sandbox/run_alaska.py

```diff
@@ -38,70 +38,67 @@
         'global_data_base/RGI_inventory/All_alaska_glacier_goodformat/')
     RGI_FILE = os.path.join(RGI_PATH,'01_rgi50_Alaska.shp')
     DATA_INPUT = os.path.join(MAIN_PATH, 'OGGM_Alaska_run/input_data/')
 
-    print(WORKING_DIR)
-    print(RGI_FILE)
-
 if Cluster:
     SLURM_WORKDIR = os.environ["WORKDIR"]
     # Local paths (where to write output and where to download input)
     WORKING_DIR = SLURM_WORKDIR
-    RGI_PATH = '~/oggm_run_alaska/input_data/Alaska_tidewater_andlake/'
+    RGI_PATH = '/home/users/bea/oggm_run_alaska/input_data/Alaska_tidewater_andlake/'
     RGI_FILE = os.path.join(RGI_PATH,'01_rgi50_Alaska.shp')
-    DATA_INPUT = '~oggm_run_alaska/input_data/'
+    DATA_INPUT = '/home/users/bea/oggm_run_alaska/input_data/'
 
-    print(WORKING_DIR)
-    print(RGI_FILE)
 
-utils.mkdir(WORKING_DIR)
+# Configuration of the run
+cfg.PARAMS['rgi_version'] = "5"
 cfg.PATHS['working_dir'] = WORKING_DIR
-
 # How many grid points around the glacier?
 # Make it large if you expect your glaciers to grow large
 cfg.PARAMS['border'] = 80
 cfg.PARAMS['optimize_inversion_params'] = True
 
 # Set to True for cluster runs
-cfg.PARAMS['use_multiprocessing'] = False
-cfg.PARAMS['continue_on_error'] = False
-
-# Don't use divides for now
-cfg.set_divides_db()
-
-# But we use intersects
+if Cluster:
+    cfg.PARAMS['use_multiprocessing'] = True
+    cfg.PARAMS['continue_on_error'] = True
+else:
+    cfg.PARAMS['use_multiprocessing'] = False
+    cfg.PARAMS['continue_on_error'] = False
+
+# We use intersects
+# (this is slow, it could be replaced with a subset of the global file)
 rgi_dir = utils.get_rgi_intersects_dir()
-rgi_shp = list(glob(os.path.join(rgi_dir, "*", '*intersects*' + rgi_reg + '_rgi50_*.shp')))
-assert len(rgi_shp) == 1
-cfg.set_intersects_db(rgi_shp[0])
-print('Intersects file: ' + rgi_shp[0])
+cfg.set_intersects_db(os.path.join(rgi_dir, '00_rgi50_AllRegs',
+                                'intersects_rgi50_AllRegs.shp'))
 
 # Pre-download other files which will be needed later
-_ = utils.get_cru_file(var='tmp')
-p = utils.get_cru_file(var='pre')
-print('CRU file: ' + p)
+utils.get_cru_cl_file()
+utils.get_cru_file(var='tmp')
+utils.get_cru_file(var='pre')
 
-# Copy the precalibrated tstar file
+# Copy the pre-calibrated t-star file
 # ---------------------------------
 # Note that to be exact, this procedure can only be applied if the run
 # parameters don't change between the calibration and the run.
 # After testing, it appears that changing the 'border' parameter won't affect
 # the results much (expectedly), so that it's ok to change it. All the rest
 # (e.g. smoothing, dx, prcp factor...) should imply a re-calibration
-mbf = 'https://www.dropbox.com/s/23a61yxwgpprs9q/ref_tstars_no_tidewater.csv?dl=1'
+mbf = 'https://github.com/OGGM/oggm-sample-data/blob/master/wgms/oggm_ref_tstars_rgi5_cru4.csv'
 mbf = utils.file_downloader(mbf)
 shutil.copyfile(mbf, os.path.join(WORKING_DIR, 'ref_tstars.csv'))
```

Comment:
This is not necessary anymore: the `distribute_tstars` task does it for you: https://github.com/OGGM/oggm/blob/master/oggm/core/climate.py#L1116
(that  is, if `cfg.PARAMS['run_mb_calibration']` is `False`, which is the default)

Reply:
So then I can just delete that whole thing?

Reply:
Yes